modules/core/mwaa/requirements/requirements.txt

```diff
@@ -1,8 +1,9 @@
 Pillow~=9.0.1
-boto3~=1.21.37
-awscli~=1.22.80
+boto3~=1.23.9
+awscli~=1.24.9
```

Comment:
the `core/mwaa` module now supports passing in a requirements.txt file to configure MWAA.  Please leverage this capability and not modify the existing code as we need to support backward-compatability